tree/30_generic_methods/file_ensure_lines_absent.cf

```diff
@@ -34,15 +34,26 @@ bundle agent file_ensure_lines_absent(file, lines)
       "args"              slist => { "${file}", "${lines}" };
       "joined"           string => join(",",args);
 
+  classes:
+      "file_exists"  expression => fileexists("${file}");
+
   files:
+    file_exists::
       "${file}"
         create        => "false",
         edit_line     => delete_lines_matching("${lines}"),
         edit_defaults => ncf_empty_select("false"),
         classes       => classes_generic_two("${old_class_prefix}", "${class_prefix}");
 
   methods:
+    file_exists::
       "sanitize" usebundle => _classes_sanitize("${class_prefix}");
       "sanitize" usebundle => _classes_sanitize("${old_class_prefix}");
       "report"   usebundle => _log("Removed content from ${file}", "${old_class_prefix}", "${class_prefix}", "${joined}");
+
+    !file_exists::
+      "success"  usebundle => _classes_success("${class_prefix}");
```

Comment:
I'm not sure about the success here (but I get why it can be useful).

Reply:
it should be _na - closest is success